stix_shifter_modules/gcp_chronicle/stix_transmission/results_connector.py

```diff
@@ -0,0 +1,374 @@
+from stix_shifter_utils.modules.base.stix_transmission.base_results_connector import BaseResultsConnector
+from stix_shifter_utils.utils.error_response import ErrorResponder
+from stix_shifter_utils.utils import logger
+import json
+from httplib2 import ServerNotFoundError
+from google.auth.exceptions import RefreshError
+import copy
+import re
+
+
+class ResultsConnector(BaseResultsConnector):
+    EMAIL_PATTERN = re.compile(r'(^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$)')
+
+    def __init__(self, api_client):
+        self.api_client = api_client
+        self.logger = logger.set_logger(__name__)
+        self.connector = __name__.split('.')[1]
+
+    def create_results_connection(self, search_id, offset, length):
+        """
+        Fetching the results using search id, offset and length
+        :param search_id: str, search id generated in transmit query
+        :param offset: str, offset value
+        :param length: str, length value
+        :return: dict
+        """
+        return_obj = {}
+        response_dict = {}
+        result = []
+        next_page_token = '0'
```

Comment:
I see `next_page_token` is hardcoded and it never gets updated. why is it always `0`?

Reply:
As discussed, updated the code to support with or without metadata in create_results_connection. IBM informed Metadata is not supported in isc-udi-connector 1.10.3.0. Once future versions supports metadata, the code should handle automatically.